src/APICalls.js

```diff
@@ -1,8 +1,7 @@
-import React from 'react'
-import moment from 'moment'
+const  rootURL = 'https://rancid-tomatillos.herokuapp.com/api/v2';
 
 export function authorizeUser(username, password) {
-  fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
+  return fetch(`${rootURL}/login`, {
```

Comment:
way to shorten this down. Wish we had some time so we could refactor the file to be one object instead of several imports though